RELEASE.md

```diff
@@ -0,0 +1,13 @@
+Release type: minor
+
+Update federation support for v2.6 which includes the @authenticated, @requiresScopes, and @policy directives.
+
+```
+    @strawberry.federation.type(
+        authenticated=True,
+        policy=[["client", "poweruser"], ["admin"]],
+        requires_scopes=[["client", "poweruser"], ["admin"]]
+    )
+    class Product(SomeInterface):
+        upc: str
+```
```

Comment:
````suggestion
This release adds support for Apollo Federation v2.6 which includes the `@authenticated`, `@requiresScopes`, and `@policy` directives.

As usual, we have first class support for them in the `strawberry.federation` namespace, here's an example:

```python
@strawberry.federation.type(
    authenticated=True,
    policy=[["client", "poweruser"], ["admin"]],
    requires_scopes=[["client", "poweruser"], ["admin"]]
)
class Product(SomeInterface):
    upc: str
```
````

Reply:
Thanks! I'll play with that test and I'll it get \@override updated to get to 2.7.  Probably will be sometime Monday.

Reply:
update to support fed spec 2.7